src/components/gcodeviewer/Viewer.vue

```diff
@@ -405,6 +401,14 @@ export default class Viewer extends Mixins(BaseMixin) {
         })
     }
 
+    get panelTitle() {
+        let title = this.$t('GCodeViewer.Title')
+
+        if (this.loadedFile) title += ': ' + this.loadedFile
```

Comment:
```suggestion
        if (this.loadedFile) title += `: ${this.loadedFile}`
```
String interpolation > String concatenation